cpp/velox/compute/WholeStageResultIterator.cc

```diff
@@ -238,17 +238,21 @@ int64_t WholeStageResultIterator::spillFixedSize(int64_t size) {
       // As of now, non-zero running threads usually happens when:
       // 1. Task A spills task B;
       // 2. Task A trys to grow buffers created by task B, during which spill is requested on task A again;
-      VLOG(2) << logPrefix << "Spill is requested on a task " << task_->taskId()
-              << " that has non-zero running threads, which is not currently supported. Skipping.";
+      LOG(WARNING) << logPrefix << "Spill is requested on a task " << task_->taskId()
+                   << " that has non-zero running threads, which is not currently supported. Skipping.";
```

Comment:
I think this case is expected but just rare. Can we keep it in INFO? We can lower the verbos info anyway.

Reply:
done